Movieee/Movieee/UIs/Home Screen/Cell/HomeScreenMovieListTableViewCell.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+class HomeScreenMovieListTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var homeScreenCollectionView: UICollectionView!
+
+    var listFilm = [CategoryResults]()
+    let idHomeCollectionView = "HomeScreenMovieListCollectionViewCell"
+    var didSelectItemAction: ((IndexPath) -> Void)?
+    
+    override func layoutSubviews() {
+        superview?.layoutSubviews()
```

Comment:
cái gì đây em?

Reply:
Em sử dụng CollectionView trong TableView, em để hết thao tác collection nằm ở tableview cell. Mà ở cell thì không có perform segue hoặc navigation, cho nên em dùng biến ni để chuyển sang Home viewcontroller đó anh

Reply:
google collection view in table view để biết cách cache index tránh bị lệch khi reuse và đưa delegate ra view controller chứ k xử lý trong cell như này nha